lib/vector-core/Cargo.toml

```diff
@@ -89,6 +89,7 @@ quickcheck_macros = "1"
 proptest = "1.1"
 similar-asserts = "1.4.2"
 tokio-test = "0.4.2"
+toml = { version = "0.7.2", default-features = false, features = ["parse"] }
```

Comment:
I'm not seeing why this has become a new dep here. It's (relatively) harmless since we're already using it elsewhere, but it would be good to keep these tidy.

Reply:
I ran into this too, seems to have come up with a recent dep update - can't run the tests in here without it (I'm not sure why).

Reply:
This is a dev dependency that a test in `vector-core` is using. Tests didn't compile directly from `vector-core` without this change. I suspect it broke a while ago and nobody noticed it until now.